wpilibc/src/main/native/cpp/Commands/ConditionalCommand.cpp

```diff
@@ -65,6 +65,14 @@ void ConditionalCommand::_Initialize() {
 
     m_chosenCommand->Start();
   }
+  m_isStarted = false;
+  Command::_Initialize();
+}
+
+void ConditionalCommand::_Execute() {
+  Command::_Execute();
```

Comment:
Shouldn't this go after the if statement?

Reply:
I don't think it matters. Why do you suggest it?

Reply:
Two reasons: the super call is after the if statement in the Java version, and what about the edge case where a command only runs for a single iteration?

Reply:
Consistency I see, but I don't understand the single iteration argument. This is calling _Execute in the ConditionalCommands base class. That is decoupled from checking the status of the chosen command.